src/pages/transaction-signing/components/post-conditions/list.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { render, waitFor } from '@testing-library/react';
+import { ProviderWithWalletAndRequestToken } from '@tests/state-utils';
+import { PostConditions } from './list';
+import { delay } from '@common/utils';
+
+const message = 'You will transfer exactly 1 HEY or the transaction will abort.';
+const from = 'From ST2P…ZE7Z';
+const firstTokenName = 'hey-token';
+const sip10Name = 'Heystack Token';
+it('has correct messaging around transfers, and uses the SIP 10 name of the heystack token', async () => {
+  const { getByText } = render(
+    <ProviderWithWalletAndRequestToken>
+      <PostConditions />
+    </ProviderWithWalletAndRequestToken>
+  );
+
+  await waitFor(() => {
+    getByText(message);
+    getByText(from);
+    getByText(firstTokenName);
+  });
+  await delay(500); // the token name is loaded async, pad the timing here
+  await waitFor(() => {
+    getByText(sip10Name);
+  });
+});
```

Comment:
```suggestion
describe('<PostConditions/ >', () => {
  it('has correct messaging around transfers, and uses the SIP 10 name of the heystack token', async () => {
    const { getByText } = render(
      <ProviderWithWalletAndRequestToken>
        <PostConditions />
      </ProviderWithWalletAndRequestToken>
    );

    await waitFor(() => {
      getByText(message);
      getByText(from);
      getByText(firstTokenName);
    });
    await delay(500); // the token name is loaded async, pad the timing here
    await waitFor(() => {
      getByText(sip10Name);
    });
  });
});
```

Reply:
Better to always wrap in a describe block, for better reporting

Reply:
done!